wagtail/wagtailadmin/wagtail_hooks.py

```diff
@@ -30,3 +31,16 @@ def register_settings_menu():
 @hooks.register('register_permissions')
 def register_permissions():
     return Permission.objects.filter(content_type__app_label='wagtailadmin', codename='access_admin')
+
+
+class PagesSearchArea(SearchArea):
+    pass
```

Comment:
Does this need to be its own subclass, or could the register line below use a plain `SeachArea`?